### PR TITLE
second pass at morango docs

### PR DESCRIPTION
**Pull request by indirectlylit**
## Read and demo

see live version here: https://new-morango-docs.netlify.app/

## Building and editing the documentation

```bash
# install requirements
pip install -r requirements/docs.txt
pip install -e .

# build docs
make docs

# build docs
make docs

# auto-build and refresh docs on edit
make docs-autobuild
```

### CODE REVIEW

File: docs/counters/index.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+IDs and Counters
+================
+
+Identifiers
+-----------
+
+There is generally one Morango instance for every Kolibri instance, and each of these are identified by a unique Morango **instance ID**. The instance ID is calculated as a function of a number of system properties, and will change when those properties change. Changes to the instance ID are not fatal, but stability is generally preferable.
+
+The **database ID** identifies the actual database being used by a Morango instance. If a database has been backed up and restored or copied to a different Morange instance, a new database ID should be generated to help other Morango instances that may have already seen the previous state of the databse.
```
Comment (bjester) [inline docs/counters/index.rst:9]: A couple misspellings here: `Morange` and `databse`

File: docs/architecture/index.rst [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    - partition string: ``${dataset_id}``
+    - used for quizzes and lessons
+
+Note that all facility models share the prefix ``${dataset_id}``, which means that they are all "contained" in that top-level partition.
+
+
+Filters and scopes
+------------------
+
+A **filter** is a set of `partition prefixes <#partitions>`__ represented as an end-line-delimited string.  A **scope** is a set of filters which defines the permissions confered by a `certificate <#certificates>`__ and stored in a ``ScopeDefinition`` object.
```
Comment (bjester) [inline docs/architecture/index.rst:70]: `confered`

File: docs/architecture/index.rst [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+- The private key associated with a parent certificate can be used to issue a child certificate to another device with at most the permission granted by the scope of the parent certificate
+- The child certificate can be used by the new device to allow it to prove to other devices that it is authorized to access the scope
+- The entire chain of signed certificates back to the origin must be exchanged during sync between devices, and the signatures and hierarchy must be verified
+
+In the example below, *Instance A* is able to establish a future sync relationship with *Instance B* by providing admin credentials to *Instance B* and requesting a signed certificate:
+
+.. image:: ./cert_exchange.png
+
+It should be cautioned that there is currenly no mechanism for revoking certificates. This means that a stolen or hijacked device will have access to all data it has been granted, and updates to that data when another device is on the same network.
```
Comment (bjester) [inline docs/architecture/index.rst:128]: `currenly`

File: docs/architecture/index.rst [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+In Kolibri, on the ``FacilityDataset`` model, we generate the certificate as a function of the ``calculate_source_id`` method. Note that we currently set the ID of the certificate to be the same as the ID of the facility model. This allows queries on the certificate hierarchy tree to find certificates that are associated with the facility.
+
+There's flexibility in the application layer for determing the validity of a root certificate, and it's specified on a per-profile basis. For the ``facilitydata`` profile, Kolibri leverages its ``auth`` models for this.
```
Comment (bjester) [inline docs/architecture/index.rst:132]: `determing`

**Comment by indirectlylit** [comment]
thanks @bjester  - pushed fixes to the typos

**Review by bjester — APPROVED** [review]
Looks good to me

**Comment by indirectlylit** [comment]
merging so I can reference in a doc I'm working on. @jamalex happy to address follow-up issues

**Comment by indirectlylit** [comment]
Doesn't seem to be rebuilding on RTD. I'll keep my netlify link up for a bit

**Pull request merged**